config/config.exs

```diff
@@ -119,7 +119,8 @@ config :skate, Skate.Repo,
 config :skate, Skate.WarmUp,
   minimum_percent_queries_to_succeed: 0.6,
   max_attempts: 20,
-  seconds_between_attempts: 1
+  seconds_between_attempts: 5,
+  backoff_min: 5_000
```

Comment:
question: I don't see any reference to this in the `WarmUp` module, what does this do?

Reply:
Oops, that's not related to the WarmUp at all, it's supposed to be part of the repo configuration! Will fix. See [here](https://hexdocs.pm/db_connection/2.4.1/DBConnection.html#start_link/2-options) for what it's actually supposed to be.